bin/live/pycbc_live_combine_single_fits

```diff
@@ -173,53 +186,63 @@ for ifo in args.ifos:
-        fout[ifo + '/daily_fits/bin_%d/date' % counter] = ad[valid_alpha]
+                                         label="%sth %%ile = %.3f" % (qstring, cons_count / meant))
         counter += 1
+
+    # Output the mean average values
+    fout[ifo + '/mean/fit_coeff'] = alphas_out[ifo]
+    fout[ifo + '/mean/counts'] = counts_out[ifo]
+
+    # Output the qth quantile values
+    fout[ifo + '/conservative/fit_coeff'] = cons_alphas_out[ifo]
+    fout[ifo + '/conservative/counts'] = cons_counts_out[ifo]
+
+    # Take some averages for plotting and summary values
     overall_invalphan = counts_out[ifo] / alphas_out[ifo]
     overall_meanalpha = counts_out[ifo].mean() / overall_invalphan.mean()
-    alpha_lines += ax_alpha.plot(ad, alpha_all,
-                                 c='k', linestyle='-', linewidth=2,
-                                 label="daily overall alpha")
-    alpha_lines += ax_alpha.plot([ad[0], ad[-1]],
-                                 [overall_meanalpha, overall_meanalpha],
-                                 c='k', linestyle='--', linewidth=2,
-                                 label="overall alpha = %.2f" % overall_meanalpha)
     sum_counts_out = counts_out[ifo].sum() / sum(live_times[ifo])
-    count_lines += ax_count.plot(ad, count_all,
-                                 c='k', linestyle='-', linewidth=2,
-                                 label="overall count")
-    count_lines += ax_count.plot([ad[0], ad[-1]],
-                                 [sum_counts_out, sum_counts_out],
-                                 c='k', linestyle='--', linewidth=2,
-                                 label="overall count per live time = %.3f" % sum_counts_out)
     save_allmeanalpha[ifo] = overall_meanalpha
-    ax_alpha.set_xlabel('Days since ' + start_date)
-    ax_alpha.set_ylabel('Fit coefficient')
-    alpha_labels = [l.get_label() for l in alpha_lines]
-    ax_alpha.legend(alpha_lines, alpha_labels, loc='lower center',
-                    ncol=3, bbox_to_anchor=(0.5, 1.01))
-    fig_alpha.tight_layout()
-    fig_alpha.savefig(args.output_plot_file_format.format(ifo + "-alphas"))
-    ax_count.set_xlabel('Days since ' + start_date)
-    ax_count.set_ylabel('Counts per live time')
-    count_labels = [l.get_label() for l in count_lines]
-    ax_count.legend(count_lines, count_labels, loc='lower center',
-                    ncol=3, bbox_to_anchor=(0.5, 1.01))
-    fig_count.tight_layout()
-    fig_count.savefig(args.output_plot_file_format.format(ifo + "-counts"))
 
-    fout[ifo + '/mean/fit_coeff'] = alphas_out[ifo]
-    fout[ifo + '/conservative/fit_coeff'] = q05_alphas_out[ifo]
-    fout[ifo + '/fixed/fit_coeff'] = [0 for a in alphas_out[ifo]]
-    fout[ifo + '/mean/counts'] = counts_out[ifo]
-    fout[ifo + '/conservative/counts'] = q95_counts_out[ifo]
+    # For the fixed version, we just 
     fout[ifo + '/fixed/counts'] = [1 for c in counts_out[ifo]]
+    fout[ifo + '/fixed/fit_coeff'] = [0 for a in alphas_out[ifo]]
```

Comment:
Many lines begin with
```
fout[ifo +
```
so I suggest creating the `ifo` group earlier and then adding stuff to it, like
```python
fout_ifo['fixed/fit_coeff'] = …
```